docs/proposals/nginx-extensions.md

```diff
@@ -1,7 +1,7 @@
 
 # Extensions for NGINX Features
 
-- Issue: #1566
+- Issue: https://github.com/nginxinc/nginx-kubernetes-gateway/issues/1566
 - Status: Provisional
```

Comment:
@kate-osborn Wasn't sure if we should update this to Implementable yet? It's a little weird since this proposal leads to more proposals.

Reply:
Implementable makes sense. I probably should've marked it as Implementable to begin with. Provisional is really for proposals that don't have any implementation details.